themes/src/main/resources-community/theme/base/login/messages/messages_pt_BR.properties

```diff
@@ -2,42 +2,51 @@ doLogIn=Entrar
 doRegister=Cadastre-se
 doCancel=Cancelar
 doSubmit=Ok
+doBack=Voltar
 doYes=Sim
 doNo=N\u00E3o
-doContinue=Continue
+doContinue=Continuar
+doIgnore=Ignorar
 doAccept=Aceitar
 doDecline=Rejeitar
-doForgotPassword=Esqueceu sua senha?
+doForgotPassword=Esqueceu a senha?
 doClickHere=Clique aqui
 doImpersonate=Personificar
+doTryAgain=Tente novamente
+doTryAnotherWay=Tente Outra Maneira
```

Comment:
By following the same pattern to other messages, I think we should have `Tente outra maneira`. Talvez, `Tente outra forma` seja mais correto ?

Reply:
Ok